components/gitpod-cli/cmd/rebuild.go

```diff
@@ -143,7 +145,7 @@ func runRebuild(ctx context.Context, supervisorClient *supervisor.SupervisorClie
 
 	tag := "gp-rebuild-temp-build"
 
-	dockerCmd := exec.Command(dockerPath, "build", "-t", tag, "--progress=tty", ".")
+	dockerCmd := exec.Command(dockerPath, "build", "-f", tmpDockerfile, "-t", tag, "--progress=tty", wsInfo.CheckoutLocation)
```

Comment:
We should not always use checkout location, but current working directory and retrieve gitpod yml from there as well, that we could check out other projects. It would be handy for ourself during dev later.

Reply:
progress=tty should be removed